ui/app/com/gu/recipeasy/views/admin/index.scala.html

```diff
@@ -4,6 +4,7 @@
     <h2>Recipeasy Admin Area</h2>
     <ul>
         <li><a href="@routes.Application.recentActivity">Recent activity</li>
+        <li><a href="@routes.Application.recentUserActivity("alice.skywalker@guardian.co.uk")">Recent activity of a specific user</li>
```

Comment:
Something just occurred to me. The default user should simply be the current user. If the current user in admin then the page will display fine (their own activity), after which they can put whichever email address they want, otherwise the page will 404.

Update coming.